src/applications/personalization/view-dependents/components/ViewDependentsFooter.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const ViewDependentsFooter = () => (
+  <div>
+    <h2 className="vads-u-font-size--h3 vads-u-padding-bottom--1 vads-u-border-bottom--3px vads-u-border-color--primary">
+      What if I have questions?
+    </h2>
+    <p>
+      You can call us at
+      <a className="nowrap" href="tel:1-800-827-1000">
```

Comment:
Should the `Telephone` component be used for this? https://design.va.gov/storybook/?path=/docs/components-telephone--default

Reply:
...yes